lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -910,6 +912,44 @@ class BuildOpGroups : public FuncOpPartialLoweringPattern {
                                  block->front().getLoc(), groupName);
   }
 
+  /// buildLibraryPipeOp will build a TCalyxLibPipeOp
```

Comment:
nitpick: I think this comment should be a sentence, like `build a TCalyxLibPipeOp, to deal with ...`